config/routes.rb

```diff
@@ -1,4 +1,9 @@
-Rails.application.routes.draw do
+  root 'users#index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
+
```

Comment:
- [ ] It would be better to have a professional README file for this project. This includes the Title and Description of the project, technologies used, how to run the app locally instructions, deployed link, screenshots, etc